# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1231822287 18000
#      Mon Jan 12 23:51:27 2009 -0500
# Node ID 4dd0213bb8835794419d0fc3154ea178c46d85d6
# Parent  5451f7460ea6b81d9e8cc40291c1672f8569119b
octave.cc (maximum_braindamage): set confirm_recursive_rmdir to false

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,10 +1,13 @@
 2009-01-12  John W. Eaton  <jwe@octave.org>
 
+	* octave.cc (maximum_braindamage): Set confirm_recursive_rmdir to
+	false.
+
 	* strfns.cc (Fstrcmp, Fstrncmp): Simplify by initializing all
 	elements of output to false with constructor argument.
 
 2009-01-12  Thorsten Meyer  <thorsten.meyier@gmx.de>
 
 	* strfns.cc (Fstrncmp): New test.
 	
 2009-01-12  John W. Eaton  <jwe@octave.org>
diff --git a/src/octave.cc b/src/octave.cc
--- a/src/octave.cc
+++ b/src/octave.cc
@@ -558,16 +558,17 @@ initialize_error_handlers ()
 static void
 maximum_braindamage (void)
 {
   persist = true;
 
   bind_internal_variable ("PS1", ">> ");
   bind_internal_variable ("PS2", "");
   bind_internal_variable ("beep_on_error", true);
+  bind_internal_variable ("confirm_recursive_rmdir", false);
   bind_internal_variable ("crash_dumps_octave_core", false);
   bind_internal_variable ("default_save_options", "-mat-binary");
   bind_internal_variable ("fixed_point_format", true);
   bind_internal_variable ("history_timestamp_format_string",
 			 "%%-- %D %I:%M %p --%%");
   bind_internal_variable ("page_screen_output", false);
   bind_internal_variable ("print_empty_dimensions", false);
 
